app/views/deep_links/mobile.html.erb

```diff
@@ -0,0 +1,42 @@
+<div id="mobile-deep-link-spinner" class="flex flex-col">
+  <svg class="crayons-icon crayons-spinner" width="24" height="24" viewBox="0 0 24 24" aria-hidden="true" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M18.364 5.636L16.95 7.05A7 7 0 1019 12h2a9 9 0 11-2.636-6.364z" fill="currentColor" class="grow-1"></path></svg>
+  <p class="align-center">Attempting to open the mobile app...</p>
+</div>
+
+<div class="mobile-deep-link-banner invisible">
+  <div class="crayons-notice crayons-notice--warning">
+    <p>
+      Whoops! Are you stuck here unable to open the mobile app?
+    </p>
+    <p class="mt-5">
+      Make sure it's installed and try again. If the problem persists contact <a href="mailto:<%= SiteConfig.email_addresses[:contact] %>"><%= SiteConfig.email_addresses[:contact] %></a>
+    </p>
+  </div>
+
+  <div class="crayons-card">
+    <div class="flex w-100">
+      <a href="javascript:history.back()" class="crayons-btn crayons-btn--l crayons-btn--secondary grow-1 whitespace-nowrap mt-5 mx-5">
+        Take me back
+      </a>
+    </div>
+
+    <div class="flex w-100">
+      <a id="link-to-mobile-install-retry" href="/" class="crayons-btn crayons-btn--l crayons-btn--secondary grow-1 whitespace-nowrap my-5 mx-5">
+        Try again
+      </a>
+    </div>
+
+    <div class="flex w-100">
+      <a id="link-to-mobile-install" href="/" class="crayons-btn crayons-btn--l grow-1 whitespace-nowrap mb-5 mx-5">
+        Install the app
+      </a>
+    </div>
+  </div>
+</div>
+
+<script type="text/javascript">
+  setTimeout(function() {
+    document.getElementById('mobile-deep-link-spinner').remove();
+    document.querySelectorAll('.mobile-deep-link-banner')[0].classList.remove('invisible');
+  }, 3333);
```

Comment:
I'm curious, why did we go with `3333` ms here? 🤔

Reply:
TBH there's no specific reason whatsoever behind this decision (100% arbitrary number)

> "Whats would be a good amount of time to let users deep link into the apps before presenting the fallback page?"

My answer to that question was "about 3 seconds, maybe" so I went with `3333` because I like the number 3 and because we never get to specify milliseconds (always do `1000/3000/5000/etc`) 😆

I can change it back to `3000` or any other number really (I have no strong opinions on this). Though it's maybe worth testing it out on our devices and see how that feels "in real life" and see if this is alright? 😄

Reply:
>  I like the number 3 and because we never get to specify milliseconds (always do 1000/3000/5000/etc)

haha, same! that's why i asked --  i never see anything beyond like 1000 or 5000 ms, and i was like "oh that's weird"

We could extract that and leave a comment that we picked that number because it's the right amount of time to let users deep link into apps before a fallback (that would make it evident that we picked it for a reason!); without that context, it feels a bit "magic numbery" to me :)